<!-- -*- mode: XML -*- -->
<launch>    


     <arg name="scen" default="scenario_1"/> 
    
     <arg name="SFDP_file_url" value="$(find SRVSS)/work_space/root.SFDP"/> 
     <arg name="resource_file_url" value="$(find SRVSS)/resource/resource_srvssDumy.xml"/> 
     <arg name="scenarios_folder_url" value="$(find SRVSS)/work_space/"/>

     <!-- setting PYTHON PATH variable to load Python Interfaces scripts -->
     <env name="PYTHONPATH" value="$(env PYTHONPATH):$(find SRVSS)/scripts"/>

     <node pkg="SRVSS" type="SamplingAlgo" name="SamplingAlgo_node" args="$(arg SFDP_file_url) $(arg scenarios_folder_url) $(arg resource_file_url)" output="screen" />


       
</launch>
